src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -288,6 +301,26 @@ const convertFrameworkDetail = async (framework) => {
       ),
     };
   }
+
+  if (launcherConfig.enabledHived) {
+    const affinityGroups = {};
+    (await axios({
+      method: 'get',
+      url: `${launcherConfig.hivedWebserviceUri}/v1/inspect/affinitygroups/`,
+      headers: launcherConfig.requestHeaders,
```

Comment:
Why need to get all groups for this single job?
Should use /v1/inspect/affinitygroups/affinityGroupName